bin/nrnivmodl.in

```diff
@@ -19,18 +19,26 @@ else
 fi
 
 if test "${NRNHOME_EXEC}" != "" ; then
-	exec_prefix="${NRNHOME_EXEC}"
 	bindir="${exec_prefix}/bin"
 	libdir="${exec_prefix}/lib"
 fi
 
+# Besides NRNHOME, one may set NRNBIN to override the default location of user binaries
+if test "x$NRNBIN" = "x"; then
+	NRNBIN=$bindir
+fi
+
 export prefix
-export exec_prefix
 export bindir
 export libdir
 
 MAKEFILEDIR="${bindir}"
 
+if which xcrun; then
+    export SDKROOT=`xcrun --sdk macosx --show-sdk-path`
+    @USING_CMAKE_TRUE@export MACOSX_DEPLOYMENT_TARGET="@CMAKE_OSX_DEPLOYMENT_TARGET@"
```

Comment:
Where does CMAKE_OSX_DEPLOYMENT_TARGET get set? A search in the sources turns up only this reference. The cmake reference states
> If not set explicitly the value is initialized by the MACOSX_DEPLOYMENT_TARGET environment variable, if set, and otherwise computed based on the host platform.

So is the purpose to get MACOSX_DEPLOYMENT_TARGET set (based on the host platform) if it was not previously set? I find it strange that that is not already the compiler default.

Reply:
Either CMake made it up, or we set in `setup.py`.
The reason why we set it explicitly is that, for wheels, we want to set it to an older version, and then we better keep matching targets